zappa/__init__.py

```diff
@@ -1,6 +1,6 @@
 import sys
 
-SUPPORTED_VERSIONS = [(2, 7), (3, 6)]
+SUPPORTED_VERSIONS = [(2, 7), (3, 6), (3, 7)]
```

Comment:
Sorry, the 3.7 update is in another PR, not merged yet. Can you reset this please.

Reply:
Ah, yep, was testing locally and, uh... may have been too lazy to explicitly call 3.6 every time. My bad!